### DISCUSSION TITLE
English spelling variants?

### DISCUSSION THREAD

**Discussion by cagenuts** [Purfview/whisper-standalone-win#167] [Q&A] [answered]
Any idea how to get the transcribing to output with non-US English? For example UK spelling. Or do I just do that in post using SE?

**Comment by Purfview** [comment, marked as answer]
Maybe such prompt would trigger it:
`-prompt="Hello, I'm from the United Kingdom. Let us use the British English spelling."`

**Comment by cagenuts** [comment]
Thanks very much, I'll give that a shot.

**Reply by Purfview** [reply]
Did it work?

**Reply by cagenuts** [reply]
I was asking for a colleague but I'll test it myself when my power returns. Thanks for checking.
